repos.d/rpm/amazonlinux.yaml

```diff
@@ -0,0 +1,33 @@
+- name: amzn2
+  type: repository
+  desc: Amazon Linux 2
+  family: fedora
+  minpackages: 2000
+  sources:
+    - name: core
+      fetcher: RepodataSqliteFetcher
+      parser: RepodataSqliteParser
+      url: http://amazonlinux.us-west-2.amazonaws.com/2/core/2.0/SRPMS/mirror.list
+  repolinks:
+    - desc: Amazon Linux 2
+      url: https://aws.amazon.com/amazon-linux-2/
+  tags: [ all, production, rpm, amazonlinux ]
+
+- name: amzn1
+  type: repository
+  desc: Amazon Linux 1
```

Comment:
Is it "1" or "AMI"? It feels like one of them should be used in both desc and repolinks.

Reply:
AMI is more consistent with the links, will fix in v2

Reply:
On second thought ... "Amazon Linux 1" and "Amazon Linux AMI" can be used interchangeably.  But "Amazon Linux 1" is more clear when listed by "Amazon Linux 2".  Will switch back main desc to "Amazon Linux 1" but will leave repo link desc to "Amazon Linux AMI".